src/doom/deh_thing.c

```diff
@@ -95,6 +95,9 @@ DEH_BEGIN_MAPPING(thing_mapping, mobjinfo_t)
   DEH_MAPPING("Action sound",        activesound)
   DEH_MAPPING("Bits",                flags)
   DEH_MAPPING("Respawn frame",       raisestate)
+  DEH_MAPPING("Melee threshold",     meleethreshold)
```

Comment:
This adds some lines to the Choco code base, so some `[crispy]` comments are due.